feature/security/gnsi/authz/tests/authz/authz1_4_test.go

```diff
@@ -46,7 +46,7 @@ const (
 type UsersMap map[string]authz.Spiffe
 
 var (
-	testInfraID = flag.String("test_infra_id", "cafyauto", "SPIFFE-ID used by test Infra ID user for authz operation")
+	testInfraID = flag.String("test_infra_id", "spiffe://test-abc.foo.bar/xyz/cafyauto", "SPIFFE-ID used by test Infra ID user for authz operation")
```

Comment:
As per Authz Proto the principals section from policy file should have absolute spiffe user id (spiffe://test-abc.foo.bar/xyz/cafyauto) or a regex based spiffe id (*/cafyauto) should be mentioned. Having plain user id "cafyauto" wont match against any of the policy rule mentioned. Hence a correction is needed here. To maintain uniformity across all the users like admin, gnoi-ping etc mentioned complete spiffe id path for cafyauto. this can be easily changed to have regex as mentioned above.